todo/src/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+
+import { ADD_TODO } from '../actions';
+
+const initialState = { todos: [] };
+
+export const reducer = (state = initialState, action) => {
+    switch (action.type) {
+        case ADD_TODO:
+            return {
+                ...state,
+                todos: state.todos.push(action.payload)
```

Comment:
```suggestion
                todos: [...state.todos, action.payload]
```
What were you doing with .push is mutating the state and we can't do that, this is will cause side effects and not rerender the list